scala-serialization/src/main/scala/com/komanov/serialization/converters/JsoniterScalaConverter.scala

```diff
@@ -0,0 +1,30 @@
+package com.komanov.serialization.converters
+
+import java.time.Instant
+
+import com.github.plokhotnyuk.jsoniter_scala.core._
+import com.github.plokhotnyuk.jsoniter_scala.macros._
+import com.komanov.serialization.domain.{Site, SiteEvent}
+
+/** https://github.com/plokhotnyuk/jsoniter-scala */
+object JsoniterScalaConverter extends MyConverter {
+  private[this] val writerConfig = WriterConfig(preferredBufSize = 131072)
```

Comment:
Such size of a buffer seems like a hack. Other libraries do reallocations during serialization/deserialization.

Reply:
@plokhotnyuk that's benchmarks-aware ad hocs
@dkomanov maybe you need to add 1Mb records ;)

Reply:
Using of preallocated thread-local buffers with the capacity that is enough for expected workload is a normal practice for any parformance-aware parser/serializer.

BTW, jsoniter-scala is able to serialize huge messages efficiently using ~32K buffers in the `writeToStream` routine:
https://github.com/plokhotnyuk/jsoniter-scala/blob/master/jsoniter-scala-core/src/main/scala/com/github/plokhotnyuk/jsoniter_scala/core/package.scala#L190